manifests/cf-manifest/env-specific/stg-lon.yml

```diff
@@ -2,3 +2,4 @@
 cell_instances: 6
 router_instances: 3
 api_instances: 2
+doppler_instances: 6
```

Comment:
I'd be inclined to drop this to 3, as we don't need the number in staging and it will still catch any issues caused during upgrade stories.

Reply:
agreed. i've made the change